### ISSUE TITLE
Tool calls missing info in accessible view, should they even be in the accessible view?

### ISSUE THREAD

**Issue by meganrogge** [microsoft/vscode#284313]
In the chat response accessible view, I see "Created" but no filename. I also see "Made changes". 

I also see "Apply patch" with just empty backticks. #284309

I'm not sure we should be putting tool calls in the accessible view. 

Let's discuss @accesswatch, @rperez030 and @jooyoungseo
